android/CMakeLists.txt

```diff
@@ -136,6 +144,7 @@ if(${FOR_HERMES})
             ${PACKAGE_NAME}
             ${LOG_LIB}
             ${HERMES_LIB}
+            ${HERMES_INSPECTOR_LIB}
```

Comment:
I believe Hermes inspector is available only in debug build so we cannot merge this in the current shape.